### ISSUE TITLE
Close Editor button on tab not readable by screenreader

### ISSUE THREAD

**Issue by kieferrm** [Microsoft/vscode#8659]
VSCode Version: insiders 7/1
OS Version: Windows

Close Editor button on editor tab not readable for screenreader.
